### ISSUE TITLE
Roadmap page

### ISSUE THREAD

**Issue by MarianRaphael** [flowforge/website#585]
### Description

**As a** FlowForge enthusiast and customer,

**I want to** see the current epic topics FlowForge is working on, as well as the potential future ones, on a dedicated page,

**And I would like to** have the possibility to vote for certain functionalities,

**So that** I can have an overview of the ongoing projects and provide feedback on the features that are most important to me.

### Acceptance Criteria

- [ ] The dedicated page should display a list of current epic topics that FlowForge is working on
- [ ] Each epic topic should have a voting button or a similar user interface element for users to express their support for that functionality.

**Comment by MarianRaphael** [comment]
First version: 
<img width="543" alt="Screenshot 2023-04-02 at 18 08 05" src="https://user-images.githubusercontent.com/73583313/229364950-c1f80b25-94da-4ffe-8753-25ed244add50.png">

**Comment by ZJvandeWeg** [comment]
Didn't know you were a front end dev too. Looks slick @MarianRaphael !

**Comment by MarianRaphael** [comment]
Query functionality: 

- Consideration – I have added a new label “consideration”. Whenever this label is assigned to an issue, it will be displayed.
- In Progress – All open epics that are assigned to an open milestone.

I have experimented with various GraphQL query alternatives, and this selection was the most appropriate based on the provided functionality set.  However, I am open to any feedback or suggestions.
